confidential-data-hub/docs/SEALED_SECRET.md

```diff
@@ -148,4 +154,10 @@ stringData:
 
 When this sealed secret is transferred to TEE, Confidential DataHub will help to unseal.
 
+## Supported Providers
+
+| Provider Name      | README                                                              | Maintainer                |
+| ------------------ | ------------------------------------------------------------------- | ------------------------- |
+| ali       	     |  [ali](kms-providers/alibaba.md)                              	   | Alibaba                   |
```

Comment:
should we use `aliyun` as the provider name to consistent with the KMS build feature?

Reply:
great. Let me fix all the typos.